docs/streams/developer-guide/dsl-api.html

```diff
@@ -3262,12 +3262,15 @@ <h5><a class="toc-backref" href="#id16">KTable-KTable Foreign-Key
 <span class="kn">import</span> <span class="nn">org.apache.kafka.streams.kstream.TimeWindows</span><span class="o">;</span>
 
 <span class="c1">// A tumbling time window with a size of 5 minutes (and, by definition, an implicit</span>
-<span class="c1">// advance interval of 5 minutes).</span>
+<span class="c1">// advance interval of 5 minutes). Note the explicit grace period, as the current</span>
+<span class="c1">// default value is 24 hours, which may be larger than needed for smaller windows. </span>
+<span class="c1">// Note that this default may change in future major version releases.</span>
```

Comment:
I'd remove the note about changing the default out. We can either find a non-breaking way to change defaults (see https://issues.apache.org/jira/browse/KAFKA-8924?focusedCommentId=17088091&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17088091 ), or we can't change it.

I'd rather not make people paranoid that we're going to break their app semantics from underneath them, because we won't.

Reply:
Kia ora @vvcephei I have removed the comment in my latest commit :)